lib/ansible/plugins/strategy/__init__.py

```diff
@@ -511,12 +511,14 @@ def parent_handler_match(target_handler, handler_name):
                                 for target_host in host_list:
                                     self._variable_manager.set_host_variable(target_host, var_name, var_value)
                         else:
+                            cacheable = result_item.pop('ansible_facts_cacheable', True)
```

Comment:
because `set_fact` is only module in which this was not true, all other 'ansible_facts' should be cacheable by default.